common/app/contentapi/ContentApiClient.scala

```diff
@@ -73,7 +73,7 @@ trait ApiQueryDefaults extends Logging {
   def item(id: String, edition: Edition): ItemQuery = item(id, edition.id)
 
   //common fields that we use across most queries.
-  def item(id: String, edition: String): ItemQuery = item(id)
+  def item(id: String, edition: String): ItemQuery = item(id.stripPrefix("/"))
```

Comment:
It would be useful to point out why the slash is being removed from the prefix, in addition to why not from the suffix as well.

Reply:
👍 will add a comment